src/App.tsx

```diff
@@ -1,26 +1,244 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { getTodos, postTodo, deleteTodo } from './api/todos';
+import { Filter } from './types/Filter';
+import { Todo as TodoType } from './types/Todo';
+import { Todo } from './components/Todo';
+import { TodoFilter } from './components/TodoFilter';
+import { ErrorMessage } from './components/ErrorMessage';
+import { TempTodo } from './components/TempTodo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todosFromServer, setTodosFromServer] = useState<TodoType[]>([]);
+  const [filterStatus, setFilterStatus] = useState<Filter>(Filter.All);
+  const [inputValue, setInputValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<null | string>(null);
+  const [lockInput, setLockInput] = useState(false);
+  const [deleteAll, setDeleteAll] = useState(false);
+
+  const [loadingError, setLoadingError] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [todoAddError, setTodoAddError] = useState(false);
+  const [deleteError, setDeleteError] = useState(false);
```

Comment:
it can be one error with appropriate error text string or empty string in case no error

Reply:
Fixed